guacamole-ext/src/main/java/org/apache/guacamole/token/StandardTokens.java

```diff
@@ -23,6 +23,8 @@
 import java.util.Date;
 import org.apache.guacamole.net.auth.AuthenticatedUser;
 import org.apache.guacamole.net.auth.Credentials;
+import java.util.Map;
+import java.util.Set;
```

Comment:
This should go above the `org.apache` entries, and be alphabetized in the correct order.